core_backend/app/question_answer/routers.py

```diff
@@ -77,6 +78,7 @@ async def llm_response(
 @identify_language__before
 @translate_question__before
 @classify_safety__before
+@classify_on_off_topic__before
```

Comment:
We also want this on embeddings-search! Shouldn't suggest top FAQs to "how much does a banana weight".